### PR TITLE
Remove articles at the beginning of sentences in crossovers

### PR DESCRIPTION
**Pull request by nzw0301**
## Motivation

~I found a few minor typos.~

As suggested by https://github.com/optuna/optuna/pull/3509#discussion_r859285858, we can remove the articles at the beginning of the first sentences in crossovers.

## Description of the changes

Remove articles.

### CODE REVIEW

File: optuna/samplers/nsgaii/_crossovers/_undx.py [hunk 1]
```diff
@@ -9,7 +9,7 @@
 
 @experimental("3.0.0")
 class UNDXCrossover(BaseCrossover):
-    """An Unimodal Normal Distribution Crossover used by :class:`~optuna.samplers.NSGAIISampler`.
+    """A Unimodal Normal Distribution Crossover used by :class:`~optuna.samplers.NSGAIISampler`.
```
Comment (himkt) [inline optuna/samplers/nsgaii/_crossovers/_undx.py:12]:
```suggestion
    """Unimodal Normal Distribution Crossover used by :class:`~optuna.samplers.NSGAIISampler`.
```

I'm wondering if we can simply remove a particle as the documentation for evaluator.
https://github.com/optuna/optuna/blob/445048a74c9090e60a82a49605044cc42727642a/optuna/importance/_mean_decrease_impurity.py#L23

File: optuna/samplers/nsgaii/_crossovers/_uniform.py [hunk 2]
```diff
@@ -5,7 +5,7 @@
 
 
 class UniformCrossover(BaseCrossover):
-    """An Uniform Crossover operation used by :class:`~optuna.samplers.NSGAIISampler`.
+    """A Uniform Crossover operation used by :class:`~optuna.samplers.NSGAIISampler`.
```
Comment (himkt) [inline optuna/samplers/nsgaii/_crossovers/_uniform.py:8]:
```suggestion
    """Uniform Crossover operation used by :class:`~optuna.samplers.NSGAIISampler`.
```

Same as https://github.com/optuna/optuna/pull/3509/files#r859285858.

**Comment by nzw0301** [comment]
@himkt Thanks! I've refect your suggestions to this PR, PR title, and PR description.

**Review by himkt — APPROVED** [review]
LGTM!

**Pull request merged**